Remotion/Validation/Core/Implementation/DiagnosticInformationValidatorFormatterDecorator.cs

```diff
@@ -110,6 +110,7 @@ private string FormatDecimalValidator (DecimalValidator validator, string typeNa
           validator.IgnoreTrailingZeros);
     }
 
+    // TODO: unused method
     private string FormatMemberInfo (MemberInfo memberInfo, Func<Type, string> typeNameFormatter)
```

Comment:
@MichaelKetting  Should I add an issue for this one or just ignore the fact that its unused?

Reply:
As you like. Personally, I'd probably just ignore it until I have to touch the method and then drop it during that update. There's just a handful of unused methods so it's not really that big of a deal.